src/sql/workbench/services/connection/browser/connectionManagementService.ts

```diff
@@ -122,6 +123,18 @@ export class ConnectionManagementService extends Disposable implements IConnecti
 
 		this._register(this._onAddConnectionProfile);
 		this._register(this._onDeleteConnectionProfile);
+
+		// keeps azure tokens refreshed so temp tables do not expire after 10 minutes
+		const seconds = 600;
+
+		setInterval(() => {
+			let connectionList = this.getActiveConnections();
+			connectionList.forEach(connection => {
+				if (connection.authenticationType === AuthenticationType.AzureMFA) {
+					this.refreshAzureAccountTokens(connection);
```

Comment:
1. the refreshAzureAccountTokens method returns a promise, you don't want to wait for it?
2. please add try catch so that one failure doesn't break the loop

Reply:
2nd comment is not addressed yet @cssuh